MapboxNavigation/Resources/Base.lproj/Localizable.strings

```diff
@@ -40,6 +40,12 @@
 /* Button title for resume tracking */
 "RESUME" = "Resume";
 
+/* Accessibility Star Label */
+"Set \(index + 1) star rating" = "Set \(index + 1) star rating";
```

Comment:
String interpolation is incompatible with localization. Use `NSString.localizedStringWithFormat(_:_:)` in conjunction with `NSLocalizedString()` and a NumberFormatter.